hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -375,6 +375,17 @@
       <artifactId>lz4-java</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>io.airlift</groupId>
+      <artifactId>aircompressor</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.hadoop.gplcompression</groupId>
+      <artifactId>hadoop-lzo</artifactId>
+      <version>0.4.21-SNAPSHOT</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Put it here for verifying the new Lzo codec only. We will remove it before merging.